vendor/src/github.com/docker/engine-api/client/image_build.go

```diff
@@ -110,6 +110,13 @@ func imageBuildOptionsToQuery(options types.ImageBuildOptions) (url.Values, erro
 		return query, err
 	}
 	query.Set("labels", string(labelsJSON))
+
+	cacheFromJSON, err := json.Marshal(options.CacheFrom)
+	if err != nil {
+		return query, err
+	}
+	query.Set("cachefrom", string(cacheFromJSON))
```

Comment:
Change them here, the vendor test will fail, once it is all worked out open
the commits on engine API, then change the vendoring to include when
committed. So dont worry now.

On 27 Aug 2016 10:46 a.m., "KJ Tsanaktsidis" <notifications@github.com>
wrote:

> In vendor/src/github.com/docker/engine-api/client/image_build.go
> <https://github.com/docker/docker/pull/24711#discussion_r76513657>:
>
> > @@ -110,6 +110,13 @@ func imageBuildOptionsToQuery(options types.ImageBuildOptions) (url.Values, erro
> >  		return query, err
> >  	}
> >  	query.Set("labels", string(labelsJSON))
> > +
> > +	cacheFromJSON, err := json.Marshal(options.CacheFrom)
> > +	if err != nil {
> > +		return query, err
> > +	}
> > +	query.Set("cachefrom", string(cacheFromJSON))
>
> What's the process around changing things in engine-api? I guess this
> needs a PR to the separate repo housing this, and a bump to the version of
> the client-daemon API.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/docker/pull/24711/files/a34650e137d63326e030c85a770c141107483a57#r76513657>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAdcPAMvCOTXiZzhloVw2s1giKZ8acHOks5qkAd1gaJpZM4JN51i>
> .
>